console/src/components/Services/Events/EventTriggers/Modify/OperationEditor.tsx

```diff
@@ -70,6 +81,15 @@ const OperationEditor = (props: OperationEditorProps) => {
       <div className={styles.modifyOpsCollapsedContent}>
         <div className={`col-md-12 ${styles.padd_remove}`}>
           Listen columns for update:&nbsp;
+          {ops.update && !readOnly ? (
+            <Button
+              color="white"
+              size="xs"
+              onClick={() => editorToggle(operationColumns)}
```

Comment:
This can be changed to :

```suggestion
              onClick={editorToggle}
```
once the above comment is rectified.